api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -1174,6 +1179,19 @@ public void maybeSendAccessExpirationEmail(DbUser user) {
     }
   }
 
+  private boolean isModuleEnabledInEnvironment(ModuleNameEnum moduleName) {
+    final AccessConfig accessConfig = configProvider.get().access;
+
+    switch (moduleName) {
```

Comment:
assuming these are the only ones that are disabled for some environments?

Reply:
Right.

Considering the feature flags:
* `enableEraCommons` - not part of AAR
* `enableBetaAccess` - not part of AAR and being removed regardless
* `enableRasLoginGovLinking` - not part of AAR

Considering the Renewable Access Modules:
* PROFILECONFIRMATION - not feature flagged
* PUBLICATIONCONFIRMATION - not feature flagged